index.html

```diff
@@ -28,7 +28,7 @@
       <!-- <div id="example"><script>include("recipes/example.md", "example")</script></div> -->
     <!-- </div> -->
     <div class="column">
-      <div id="example"><script>include("recipes/DragonPaneerChilli.md", "Dragon Paneer Chilli")</script></div>
```

Comment:
Please change here `"example"` to `"Dragon"` and `"Dragon Paneer Chilli"` to `"Dragon"`